src/pages/LandingPage.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import NavBarOut from '../components/NavBarOut';
+
+const LandingPage = () => <NavBarOut />;
```

Comment:
this component seems pointless....just render navbarout in your rourter instead of landing page